examples/interactionsDemo.html

```diff
@@ -0,0 +1,526 @@
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css" integrity="sha384-aUGj/X2zp5rLCbBxumKTCw2Z50WgIr1vs/PFN4praOTvYXWlVyh2UtNUU0KAUhAX" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+            crossorigin="anonymous">
+        <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+        <script src="https://sdk-cdn.mypurecloud.com/client-apps/1.0.0/purecloud-client-app-sdk-ed1f20f5244ee2bb9cc242f669369639.min.js"></script>
+        <script src="https://sdk-cdn.mypurecloud.com/javascript/27.0.0/purecloud-platform-client-v2.min.js"></script>
+
+        <style>
+            body {
+                margin-left: 20px;
+                max-width: 450px;
+            }
+
+            #submitConversationIDButton {
+                margin-top: 10px;
+                justify-content: flex-end;
+            }
+
+            .hoverRow:hover {
+                cursor: pointer;
+            }
+
+            .topGroup {
+                display: flex;
+            }
+
+            #reloadButton {
+                margin-left: auto;
+                height: 63px;
+            }
+
+            .failure {
+                color: red;
+            }
+
+            .reloading {
+                margin: 10px;
+                text-align: center;
+                width: 100%;
+            }
+
+            .pulse {
+                display: inline-block;
+                width: 10px;
+                height: 10px;
+                border-radius: 50%;
+                background: red;
+                box-shadow: 0 0 0 rgba(204, 44, 44, 0.4);
+                animation: pulse 2s infinite;
+            }
+
+            @keyframes pulse {
+                0% {
+                    box-shadow: 0 0 0 0 rgba(204, 44, 44, 0.4);
+                }
+                70% {
+                    box-shadow: 0 0 0 10px rgba(204, 44, 44, 0);
+                }
+                100% {
+                    box-shadow: 0 0 0 0 rgba(204, 44, 44, 0);
+                }
+            }
+        </style>
+    </head>
+
+    <body>
+        <noscript>
+            For full functionality of this site it is necessary to enable JavaScript. Here are the <a href="http://www.enable-javascript.com/" target="_blank">instructions how to enable JavaScript in your web browser</a>.
+        </noscript>
+
+        <h1>Show Interaction Details (Past 30 days)</h1>
+
+        <section class="authenticating hidden">
+            Authenticating to PureCloud <span class="fas fa-spinner fa-spin"></span>
+        </section>
+        <section class="failure hidden">
+            Sorry, an error has occurred.
+        </section>
+
+        <div class="topGroup">
```

Comment:
Ideally, this section would only show once the initial setup and auth is done.  It can/should be shown in the fetch failure case though so that the user can retry.